easybuild/easyconfigs/c/CESM-deps/CESM-deps-2-foss-2022a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'CESM-deps'
+version = '2'
+
+homepage = 'https://www.cesm.ucar.edu/models/cesm2/'
+description = """CESM is a fully-coupled, community, global climate model that
+provides state-of-the-art computer simulations of the Earth's past, present,
+and future climate states."""
+
+# The following environment is suitable for CESM >= 2.2.0
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('CMake', '3.23.1'),
+    ('Python', '3.10.4'),
+    ('lxml', '4.9.1'),
+    ('Perl', '5.34.1'),
+    ('XML-LibXML', '2.0207'),
+    ('ESMF', '8.3.0'),
+    ('netCDF', '4.9.0'),
+    ('netCDF-Fortran', '4.6.0'),
+    ('netCDF-C++4', '4.3.1'),
+    ('PnetCDF', '1.12.3'),
+    ('Subversion', '1.14.2'),
+    ('git', '2.36.0', '-nodocs'),
+    ('git-lfs', '3.2.0', '', SYSTEM),
+]
+
+components = [
+    # install extra configuration tools and files for VSC clusters
+    ('cesm-config', '1.6.2', {
+        'easyblock': 'Tarball',
+        'source_urls': ['https://github.com/vub-hpc/%(name)s/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'checksums': ['eda865c6c6885253728de0d570a850a728310785162c8b78bd896b0d1dfbbf48'],
+        'start_dir': '%(name)s-%(version)s',
+    }),
+]
```

Comment:
@lexming is this part suitable for the upstream repo, or should it go in a hook for your site?

Reply:
It does not hurt to install it, it just contains some configuration files and a script to update configuration files. It was already part of `CESM-deps-2-foss-2021b.eb`